posthog/urls.py

```diff
@@ -38,7 +38,7 @@
 from .views import health, preflight_check, stats, system_status
 
 
-def home(request, **kwargs):
+def home(request, *args, **kwargs):
```

Comment:
At some point changed the unauthenticated route handling to use `re_path` instead of `path` to support more complex routing. `re_path` sends a second positional argument, so `*args` is used to catch it. I reverted those changes in a later commit, I can remove this but I think it can be useful if we do decide to go back to `path`